examples/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "examples",
+  "version": "0.0.0",
+  "private": true,
+  "type": "module",
+  "exports": {
+    ".": "./src/index.tsx",
+    "./*": "./*"
+  },
+  "scripts": {
+    "build": "tsup src/index.tsx"
+  },
+  "devDependencies": {
+    "tsup": "^6.7.0"
+  }
```

Comment:
remove `tsup`, not needed anymore
```suggestion
```

Reply:
Fixed!

Reply:
run `yarn` and push the updated lockfile